pkgs/top-level/all-packages.nix

```diff
@@ -24497,7 +24497,9 @@ in
 
   thepeg = callPackage ../development/libraries/physics/thepeg { };
 
-  yoda = callPackage ../development/libraries/physics/yoda { };
+  yoda = callPackage ../development/libraries/physics/yoda {
+    pythonPackages = python2Packages;
```

Comment:
python2 is EOL, please use python3Packages if possible

Reply:
using my other suggestions:
```suggestion
    python = python3;
```

Reply:
I will be switching `yoda` to use `python3` by default, but that, probably, can not be done until I get `rivet` version 3 working.